mon-pix/translations/en.json

```diff
@@ -963,6 +963,22 @@
           "url-text": "Contactez le support",
           "recover": " pour récupérer votre compte Pix."
         }
+      },
+      "confirmation-step": {
+        "good-new" : "Bonne nouvelle { firstName } !",
```

Comment:
**chipotage**
News est singulier (pas comme français), même dans " a piece of news" :teacher: 
#IAmTheNewEmmy
```suggestion
        "good-news" : "Bonne nouvelle { firstName } !",
```